generate_site/index_source.html

```diff
@@ -34,7 +34,10 @@
     <nav class="navbar navbar-inverse" role="navigation" style="padding-bottom:0px; border-color:#4a423d!important; background-color:#4a423d!important;">
       <div class="container" style="padding-bottom:5px; padding-top:5px; ">
         <div class="navbar-header">
-          <a class="navbar-brand" href="#Monitor">Monitor wariantów SARS-COV-2 w Polsce</a>
+          <a class="navbar-brand" href="#Monitor">Monitor wariantów SARS-CoV-2 w Polsce</a>
+          <div class="navbar-brand">
+            <a class="navbar-link" href="/">PL</a> / <a class="navbar-link" href="/en">EN</a>
```

Comment:
Absolute paths "/" or "/en" will fail on the development server and in subfoltders for different countries. I would rather use `href="index.html"` and `href="index_en.html"`

Reply:
Let's make it `index.html` and `en.html`